src/scrabbag.js

```diff
@@ -0,0 +1,50 @@
+//input a string
+//split string into an array
+//iterate over the array
+//for each letter and subtract one
+//MAKE THE STRING UPPERCASE
+
+var scrabbleBag = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+  _: 2
+}
+
+function scrabbleRemainder(str){
+var upperCase = str.toUpperCase()
```

Comment:
I try to avoid the declaration of variables that will only be used once - it's an extra operation/memory allocation, and in cases like this it could be more concisely expressed inline:
```js
const array = input.toUpperCase().split( ' ' )
```
Note also that I renamed the variables - `str` is not a meaningful, intention revealing name in the context of this function.  As well, `toArray` describes an action, not a piece of data - I interpreted it briefly as a function rather than a variable.